src/components/PathwaysList/PathwaysList.tsx

```diff
@@ -92,6 +105,21 @@ const PathwaysList: FC<PathwaysListProps> = ({ evaluatedPathways, callback, serv
     );
   }
 
+  const getSelectedPathways = (): string[] => {
+    // Get all active CarePlan resource texts
+    const carePlanTexts = (resources.filter(r => r.resourceType === 'CarePlan') as CarePlan[])
+      .filter(r => r.status === 'active')
+      .map(r => r.text?.div);
```

Comment:
Is there a reason you chose text here over title? I feel though title is more appropriate so instead of checking the text includes the pathway name you just check the title is the same as the pathway name. Both work but title seems more natural to me

Reply:
Just to clarify some more after reviewing the createCarePlan function - we should still leave the text explaining what it is but also setting the title will make for an easier comparison than checking the pathway name is included in the text

Reply:
yea I actually had it checking the title at the beginning of this PR, but I changed the CarePlan to conform with USCore profile which required the text property. Since we were including the pathway name in the text property I decided to just check for it there. It's actually easier to check for it in the title so I'm not opposed to adding the title property back. Any preference @dehall?

Reply:
Yeah that sounds good. Keep text as "assignment of patient to pathway {x}" and then title is just the pathway name

Reply:
Cool, this change has been pushed.